tools/build-release-tarball

```diff
@@ -69,6 +69,8 @@ mv zulip-git-version "$TMPDIR/$prefix/"
 
 cd "$TMPDIR/$prefix"
 
+ln -s "$BASEDIR/zulip-py3-venv" .
+
```

Comment:
We probably want to do this regardless of the rest, for consistency?  Could go in a prep commit.